samples/client/petstore/python-experimental/docs/FakeApi.md

```diff
@@ -682,13 +682,13 @@ except ApiException as e:
 
 Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
- **enum_header_string_array** | [**list[str]**](str.md)| Header parameter enum test (string array) | [optional]
```

Comment:
@spacether looks like list has been incorrectly removed from the doc

Reply:
@wing328 the description at the right describes this parameter as 
`[str]` which is a list of string. What specifically do you see being removed?
If it is the word list, we now use [] to represent that, which is needed because the [str] the the content that is inserted when defining the type of enum_header_string_array in fake_api.py:
```
                'openapi_types': {
                    'enum_header_string_array': ([str],),
```

Reply:
Please let me know if that resolves your question.

Reply:
Ah ok. Thanks for the explanation.